openpype/modules/deadline/plugins/publish/submit_3dmax_deadline.py

```diff
@@ -0,0 +1,237 @@
+
+            metadata_folder = instance.data.get("publishRenderMetadataFolder")
+            if metadata_folder:
+                metadata_folder = metadata_folder.replace(orig_scene,
+                                                          new_scene)
+                instance.data["publishRenderMetadataFolder"] = metadata_folder
+
+        payload = {
+            "JobInfo": {
+                # Top-level group name
+                "BatchName": filename,
+
+                # Job name, as seen in Monitor
+                "Name": jobname,
+
+                # Arbitrary username, for visualisation in Monitor
+                "UserName": deadline_user,
+
+                "Plugin": instance.data["plugin"],
+                "Group": self.group,
+                "Pool": self.deadline_pool,
+                "secondaryPool": self.deadline_pool_secondary,
+                "Frames": frames,
+                "ChunkSize": self.chunk_size,
+                "Priority": instance.data.get("priority", self.priority),
+                "Comment": comment,
+                "FramesPerTask": self.framePerTask
+            },
+            "PluginInfo": {
+                # Input
+                "SceneFile": filepath,
+                "Version": "2023",
+                "SaveFile": True,
+                # Mandatory for Deadline
+                # Houdini version without patch number
+
+                "IgnoreInputs": True
+            },
+
+            # Mandatory for Deadline, may be empty
+            "AuxFiles": []
+        }
+        # Include critical environment variables with submission + api.Session
+        keys = [
+            # Submit along the current Avalon tool setup that we launched
+            # this application with so the Render Slave can build its own
+            # similar environment using it, e.g. "maya2018;vray4.x;yeti3.1.9"
+            "AVALON_TOOLS",
+            "OPENPYPE_VERSION"
+        ]
+        # Add mongo url if it's enabled
+        if context.data.get("deadlinePassMongoUrl"):
+            keys.append("OPENPYPE_MONGO")
+
+        environment = dict({key: os.environ[key] for key in keys
+                            if key in os.environ}, **legacy_io.Session)
+
+        payload["JobInfo"].update({
+            "EnvironmentKeyValue%d" % index: "{key}={value}".format(
+                key=key,
+                value=environment[key]
+            ) for index, key in enumerate(environment)
+        })
+
+        # Include OutputFilename entries
+        # The first entry also enables double-click to preview rendered
+        # frames from Deadline Monitor
+        output_data = {}
+        # need to be fixed
+        for i, filepath in enumerate(instance.data["expectedFiles"]):
+            dirname = os.path.dirname(filepath)
+            fname = os.path.basename(filepath)
+            output_data["OutputDirectory%d" % i] = dirname.replace("\\", "/")
+            output_data["OutputFilename%d" % i] = fname
```

Comment:
I think that we are doing this wrong here - and not just here but with Maya too - If I am not mistaken, `OutputFilename` can take as value sequence - so instead of one `OutputFilename` per file in `expectedFiles` we should set one OutputFilename per sequence, just replacing frame number with `#`